release_bot/utils.py

```diff
@@ -252,3 +253,26 @@ def update_version(file, new_version):
             output.write('\n'.join(content) + '\n')
         configuration.logger.info('Version replaced.')
     return changed
+
+
+def parse_setupcfg():
+
+    """
+    Getting the data from the metadata section
```

Comment:
The function name and description should reflect that it returns only the name (not all metadata or even whole config)

Reply:
it could return all the data so maybe i should change what it returns and not the name, i mean it will be waste not to

Reply:
does not seem to be resolved